mailbox/store/src/main/java/org/apache/james/mailbox/store/event/MailboxAnnotationListener.java

```diff
@@ -34,11 +36,13 @@
 
 public class MailboxAnnotationListener implements MailboxListener {
     private static final Logger logger = LoggerFactory.getLogger(MailboxAnnotationListener.class);
-    private MailboxSessionMapperFactory mailboxSessionMapperFactory;
+    private final MailboxSessionMapperFactory mailboxSessionMapperFactory;
+    private final MailboxManager mailboxManager;
 
     @Inject
```

Comment:
is it ever injected ?

Reply:
It is, I checked by debugging integration tests :)